retail/retail-clickstream-application/README.md

```diff
@@ -0,0 +1,234 @@
+# Click stream data - Data Layer to Pub/Sub
+
+![Cloud Run Proxy](cloud-run-proxy.png)
+
+This repo contains example code to show how to send data from a ecommerce website to Pub/Sub using Google Tag Manager, Cloud Run and Pub/Sub.
```

Comment:
I think we need a little more info on what the solution looks like in terms of architecture. You should also have some notes on : If you would like to setup a end to end environment then you can set this up. "Or you can use this as samples, here is a list of links to the code for each piece..."

Reply:
I have added a new diagram and a better description.Please let me know if this is ok. Not sure how to link to each section as not everything has a folder